client/util/util.go

```diff
@@ -40,6 +41,7 @@ type Libs struct {
 // possibly due to entire config missing or missing binary path.
 var ErrConfigUnavailable = errors.New("Config is unavailable")
 
+/* UPDATED FUNCTION */
```

Comment:
It's okay to remove this. We can use the "diff" to see what changes you have made.

Reply:
got it!